config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/v1/models/PackageConfigListApiModel.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.azure.iotsolutions.uiconfig.webservice.v1.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.microsoft.azure.iotsolutions.uiconfig.services.models.PackageConfigType;
+import com.microsoft.azure.iotsolutions.uiconfig.services.models.PackageConfigurations;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class PackageConfigListApiModel {
+
+    @JsonProperty("configurations")
+    public String[] packageConfigurations;
+
+    public String[] getPackageConfigurations()
+    {
+        return packageConfigurations;
+    }
+
+    public PackageConfigListApiModel(PackageConfigurations packageConfigurations)
+    {
+        List<String> customConfigs = new ArrayList<String>(Arrays.asList(packageConfigurations.configurations));
```

Comment:
arrays.asList already returns a list<string> do you need to create an arraylist?

Reply:
also same comment as .net that customConfigs is a confusing variable name

Reply:
Thanks, changed.